documentation/sdn/ovn_bgp_evpn.md

```diff
@@ -0,0 +1,125 @@
+
+- **Scalability and multi-tenant environments** - The OVN BGP Agent enables scalable multi-tenant environments. Each tenant's network can be advertised separately via BGP, and policies can be applied based on BGP advertisements. This makes it easier to manage large-scale cloud environments where multiple tenants or customers need isolated but externally reachable networks. No manual configuration of floating IPs and NATs required.
+- **Enhanced security** - BGP routing policies can be used to apply access controls, such as filtering routes or controlling which networks can be advertised and accepted. This allows for more granular control of how traffic flows between the virtualized environment and external networks. Without OVN BGP Agent, security policies would need to be applied manually or through static firewall and NAT configurations, which may not be as flexible or scalable as BGP-based route filtering and access control mechanisms.
+
+The OVN BGP Agent can be used by cloud providers or large enterprises to allow external access to VMs and containers without relying on NAT.
+
+## EVPN
+
+Technology used for extending Layer 2 (L2) networks over Layer 3 (L3) networks,
+typically in data center and WAN environments. EVPN allows multiple sites, such
+as data centers or remote locations, to appear as if they are part of the same
+L2 network, even though they are connected over a L3 IP network.
+
+It is a standardized extension of BGP, more precisely of Multiprotocol BGP
+(MP-BGP), which is used to advertise MAC addresses (L2) and IP prefixes (L3)
+together with additional VPN information. This removes the need of flooding as
+a method for MAC discovery and leads to more efficient L2 networking. Using BGP
+as a single protocol for both L2 and L3 advertisements results in simplified
+management with only one control plane.
+
+Additionally, EVPN supports anycast gateways, allowing multiple routers to
+provide L3 services with the same IP address, improving redundancy and
+failover. It also enables seamless VM live migration across geographically
+distributed environments, preserving the state of L2 and L3 connectivity.
+
+VXLAN or MLSP are used as data plane, to deliver the L2 frames over the L3
+underlay. Each tenant’s traffic is encapsulated in VXLAN tunnels between
+endpoints (known as VTEPs—VXLAN Tunnel Endpoints), and BGP advertises the VXLAN
+VNI (Virtual Network Identifier) mappings along with MAC/IP addresses. As an
+alternative MPLS (Multi Protocol Label Switching) is well-suited for situations
+where low-latency and highly predictable paths are required, or where MPLS
+infrastructure already exists.
+
+Geneve tunnels, VLANs or VXLANs can be also used by Neutron as ML2 drivers to
```

Comment:
Multitenancy is the de facto standard (never came across even a private cloud that doesn't have tenant networks, whether that be VLAN-backed or VXLAN/Geneve). Maybe the "can also be used" is an underestimation ;-)